.github/workflows/root-ci.yml

```diff
@@ -187,13 +190,6 @@ jobs:
           name: Test Results ${{ matrix.platform }} ${{ matrix.arch }}
           path: /Users/sftnight/ROOT-CI/build/TestResults.xml
 
-      - name: Upload build artifacts
```

Comment:
Why is this section removed?

Reply:
It is an exact duplicate of the code above (line 183 to 189)

Reply:
It is an almost exact replicate of the code above (line 183-188) where only the 'name' is different.